pulp_rpm/app/management/commands/rpm-datarepair.py

```diff
@@ -0,0 +1,38 @@
+from gettext import gettext as _
+
+from django.core.management import BaseCommand
+
+from pulp_rpm.app.models import Package  # noqa
+
+
+class Command(BaseCommand):
+    """
+    Django management command for repairing RPM metadata in the Pulp database.
+    """
+
+    help = _(__doc__)
+
+    def add_arguments(self, parser):
```

Comment:
Should we pre-emptively assume we're going to end up extending this command and have a required issue #?

Reply:
It has a pretty generic name, so I can def see wanting to extend its functionality if/when we hit the **next** metadata-cleanup problem. I tink you're right, we prob should plan for the future here.